docs/source/techniques/ISISPowder-Polaris-v1.rst

```diff
@@ -400,6 +402,20 @@ Example Input:
 
   polaris_example.focus(do_van_normalisation=True, ...)
 
+.. _empty_can_subtraction_method_isis-powder-diffraction-ref:
+
+empty_can_subtraction_method
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Sets the absorption method used in the :ref:`focus_polaris_isis-powder-diffraction-ref` method. This parameter
+is optional and can be set to either ``"Simple"`` (default) or ``"PaalmanPings"``. In ``"PaalmanPings"`` mode
+the :ref:`paalman_pings_events_per_point_isis-powder-diffraction-ref` parameter can be utilised.
```

Comment:
Perhaps worth mentioning here that the PaalmanPings option results in the PaalmanPingsMonteCarloAbsorption algorithm being run followed by ApplyPaalmanPingCorrection